src/main/java/org/opensearch/ad/constant/CommonErrorMessages.java

```diff
@@ -73,8 +75,10 @@ public static String getTooManyCategoricalFieldErr(int limit) {
     public static String HISTORICAL_ANALYSIS_CANCELLED = "Historical analysis cancelled by user";
     public static String HC_DETECTOR_TASK_IS_UPDATING = "HC detector task is updating";
     public static String NEGATIVE_TIME_CONFIGURATION = "should be non-negative";
+    public static String INVALID_TIME_CONFIGURATION_UNITS = "Timezone %s is not supported";
```

Comment:
The error message seems not correct from the first release. Change to "Time unit %s is not supported" ?

Reply:
good catch! Changed this.